src/DGtal/base/Alias.h

```diff
@@ -199,61 +199,67 @@ user forward an Alias<T> parameter.
     inline ~Alias() {}
 
     /**
-       Constructor from const reference to an instance of T. Invalid.
+       Constructor from const reference to an instance of T. Deleted.
+
+       Aliasing a const-ref is an error. Consider ConstAlias instead.
     */
-    inline Alias( const T& ) : myParam( CONST_LEFT_VALUE_REF ), myPtr( 0 )
-    { ASSERT(( false && "[Alias::Alias( const T& )] Aliasing a const-ref is an error. Consider ConstAlias instead." )); }
+    Alias( const T& ) = delete;
```

Comment:
Actually, I was thinking about deleting all those special members, since the copy operations are currently forbidden in the templates (see e.g. https://github.com/DGtal-team/DGtalScripts/blob/master/dev/models/TXXX.h)

It will give something like:
```
public:
XXX() = delete;
~XXX() = delete;
XXX( const XXX & ) = delete;
XXX( XXX && ) = delete;
XXX & operator= ( const XXX & ) = delete;
XXX & operator= ( XXX && ) = delete;
```
with appropriate documentation for each ones, and then letting the developer choosing between:
- keeping it deleted,
- switching it to the default behavior by replacing `delete` with `default`,
- implementing his own operations.